x/bank/internal/keeper/keeper_test.go

```diff
@@ -194,6 +194,95 @@ func TestMsgSendEvents(t *testing.T) {
 	require.Equal(t, event2, events[3])
 }
 
+func TestMsgMultiSendEvents(t *testing.T) {
+	app, ctx := createTestApp(false)
+
+	app.BankKeeper.SetSendEnabled(ctx, true)
+
+	addr := sdk.AccAddress([]byte("addr1"))
+	addr2 := sdk.AccAddress([]byte("addr2"))
+	addr3 := sdk.AccAddress([]byte("addr3"))
+	addr4 := sdk.AccAddress([]byte("addr4"))
+	acc := app.AccountKeeper.NewAccountWithAddress(ctx, addr)
+	acc2 := app.AccountKeeper.NewAccountWithAddress(ctx, addr2)
+
+	app.AccountKeeper.SetAccount(ctx, acc)
+	app.AccountKeeper.SetAccount(ctx, acc2)
+	newCoins := sdk.NewCoins(sdk.NewInt64Coin("foocoin", 50))
+	newCoins2 := sdk.NewCoins(sdk.NewInt64Coin("barcoin", 100))
+	inputs := []types.Input{
+		{addr, newCoins},
```

Comment:
Please fix this warning

Reply:
Is it better to add field keys or to use types.NewInput(...) function?

Reply:
It's OK to just use the field names 👍